app/views/directives/_service-binding.html

```diff
@@ -46,7 +46,7 @@
       project-name="{{$ctrl.binding.metadata.namespace}}"
       stay-on-current-page="true">
     </delete-link>
-    <a ng-if="('secrets' | canI : 'get') && !($ctrl.binding | isBindingFailed) && !(binding | isBindingReady)"
+    <a ng-if="('secrets' | canI : 'get') && ($ctrl.binding | isBindingReady)"
```

Comment:
@jeff-phillips-18 The failed check isn't necessary because the binding is never ready when the `Failed` condition is true. (Checked with @pmorie on this a few days ago.)

Reply:
Note that the logic before was `!(binding | isBindingReady)`

Reply:
Thanks @spadgett